guides/source/api_app.md

```diff
@@ -105,7 +105,7 @@ Handled at the Action Pack layer:
   for three kinds of HTTP authentication.
 - Instrumentation: Rails has an instrumentation API that triggers registered
   handlers for a variety of events, such as action processing, sending a file or
-  data, redirection, and database queries. The payload of each event comes with
+  data, redirection and database queries. The payload of each event comes with
```

Comment:
According to our docs on documentation the Oxford comma is preferred. See http://guides.rubyonrails.org/v5.0/api_documentation_guidelines.html#oxford-comma